src/index.html

```diff
@@ -9,10 +9,46 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/index.css"
     />
   </head>
+
   <body>
     <h1>Calendar</h1>
+    <div class="block_calendar">
+      <div class="calendar calendar--month-length-31 calendar--start-day-sun">
```

Comment:
The class names 'calendar--month-length-31' and 'calendar--start-day-sun' suggest that you are using a modifier in BEM methodology, but it is not clear what element or block is being modified. Ensure that your BEM notation correctly reflects the elements, blocks, and modifiers.